src/Blocks/Zaken/Block.php

```diff
@@ -129,6 +128,9 @@ protected function enrichZaak(Zaak $zaak, Client $client): Zaak
         $zaak->setValue('leverancier', $client->getClientName());
         $zaak->setValue('steps', is_object($zaak->zaaktype) && $zaak->zaaktype->statustypen instanceof Collection ? $zaak->zaaktype->statustypen->sortByAttribute('volgnummer') : []);
         $zaak->setValue('status_history', $zaak->statussen);
+        $zaak->setValue('information_objects', $zaak->zaakinformatieobjecten);
```

Comment:
Hier ook `$zaak->informationObjects()` ipv `$zaak->zaakinformatieobjecten`?

Reply:
Nee dit heeft een andere implementatie `$zaak->zaakinformatieobjecten` roept de magic method `__get` aan die weer kijkt naar de casts. (gedefinieerd in de entity)